modules/default/calendar/calendar.js

```diff
@@ -232,7 +234,10 @@ Module.register("calendar", {
 					const dateRow = document.createElement("tr");
 					dateRow.className = "dateheader normal";
 					if (event.today) dateRow.className += " today";
+					else if (event.dayBeforeYesterday) dateRow.className += " dayBeforeYesterday";
+					else if (event.yesterday) dateRow.className += " yesterday";
 					else if (event.tomorrow) dateRow.className += " tomorrow";
+					else if (event.dayAfterTomorrow) dateRow.className += " dayAfterTomorrow";
```

Comment:
Can you add tests for these classnames in tests/electron/modules/calendar_spec.js ?

Reply:
Hi, i will try to. I struggled with the last time i opened a pull request and failed but this time new chance.

Reply:
I added some checks to prove that the CSS classes got added correctly which was more or less copy and paste. I will try to add more changes but i will need to dive deeper into into the test classes to check the locals